doc/source/generation.rst

```diff
@@ -98,21 +114,32 @@ Random graphs
 +++++++++++++
 Stochastic graphs can be created according to several different models or games:
 
- - bipartite :meth:`Graph.Random_Bipartite`
  - Barabasi-Albert model: :meth:`Graph.Barabasi`
  - Erdos-Renyi: :meth:`Graph.Erdos_Renyi`
  - Watts-Strogatz :meth:`Graph.Watts_Strogatz`
+ - stochastic block model :meth:`Graph.SBM`
  - forest fire game :meth:`Graph.Forest_Fire`
  - random geometric graph :meth:`Graph.GRG`
  - growing :meth:`Graph.Growing_Random`
  - establishment game :meth:`Graph.Establishment`
  - preference, the non-growing variant of establishment :meth:`Graph.Preference`
  - asymmetric preference :meth:`Graph.Asymmetric_Prefernce`
  - recent degree :meth:`Graph.Recent_Degree`
- - non-growing graph with edge probabilities proportional to node
-fitnesses :meth:`Graph.Static_Fitness`
+ - K-regular (each node has degree K) :meth:`Graph.K_Regular`
+ - non-growing graph with edge probabilities proportional to node fitnesses :meth:`Graph.Static_Fitness`
  - non-growing graph with prescribed power-law degree distributions :meth:`Graph.Static_Power_Law`
  - in agreement with a sequence of degrees :meth:`Graph.Degree_Sequence`
+ - bipartite :meth:`Graph.Random_Bipartite`
+
 
+Other graphs
+++++++++++++
+Finally, there are some ways of generating graphs that are not covered by the previous sections:
 
+ - Kautz graphs :meth:`Graph.Kautz`
+ - De Bruijn graphs :meth:`Graph.De_Bruijn`
+ - Lederberg-Coxeter-Frucht graphs :meth:`Graph.LCF`
+ - graphs with a specified isomorphy class :meth:`Graph.Isoclass`
+ - 
```

Comment:
There's an extra bullet point here

Reply:
"isomorphism", not "isomorphy" (apparently that's biology, not math)